apps/admin-gui/src/assets/config/version.json

```
@@ -0,0 +1 @@
+{"version":"0.10.3"}
```

As I mentioned in a different comment, this file should be either handled manually or maybe a better solution would be to put in the version some information, that this is a test version. 
```suggestion
{"version":"SNAPSHOT"}
```
Maybe something like this should work. That way, we don't need to manually bump the version each time a new version was released.

---

Please, change this version to the suggested value.